machine.go

```diff
@@ -288,6 +295,10 @@ func (m *Machine) setError(err error) {
 	m.err = err
 	m.mu.Unlock()
 	m.setState(Stopped)
+	m.event("bigmachine:machineError",
```

Comment:
What about just having `bigmachine` be autoprefixed by event? Are you wanting to use this as a layer for bigslice also?

Reply:
For this first pass, I'm trying to keep it as simple as possible. There are fewer than a dozen events to log right now, so I don't think there's much lost by just repeating ourselves. If usage grows, we'll definitely have to add more structure/schema to these keys, but I'm inclined to keep it pretty loose until it starts to feel painful.